app/views/home/index.html.erb

```diff
@@ -1 +1,20 @@
-<%= link_to("log out", destroy_admin_session_path, method: :delete) %>
\ No newline at end of file
+<nav class="navbar navbar-dark navbar-expand-lg bg-dark flex-md-nowrap p-0 shadow">
```

Comment:
Hi Andrea you use the class "navbar-expand-lg" for that reason I think is necessary a button that collapses the content behind to hide or show it.
```suggestion
<nav class="navbar navbar-dark navbar-expand-lg bg-dark flex-md-nowrap p-0 shadow">
<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
```
Or for a nav always expanded
```suggestion
<nav class="navbar navbar-dark navbar-expand bg-dark flex-md-nowrap p-0 shadow">
```

Reply:
I think its better to use nav always expanded instead of collapse it, mainly because we will need a sidebar, and in my opinion, sidebar would be better if it can collapse.
WDYT?

Reply:
I agree with you, just that usually the button to collapse sidebar is on navbar, the code doesn't define a navbar always expanded, define a navbar expanded on large screens "navbar-expand-lg"